streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -202,6 +206,8 @@ public void testProcessOrder() throws Exception {
         assertEquals(0, task.numBuffered());
         assertEquals(3, source1.numReceived);
         assertEquals(3, source2.numReceived);
+
+        cleanRun = true;
```

Comment:
Need to double check. Will reply here or just update if we can simplify the code.